src/components/views/ProfilePage.jsx

```diff
@@ -0,0 +1,181 @@
+                          </div>
+                        </div>
+                        <p className="profilebox-user__bio">
+                          {profile.bio}
+                        </p>
+                      </div>
+                    </div>
+                    <div className="column" />
+                    <div className="column">
+                      <div className="profilebox-users">
+                        <div className="profilebox-user__followers">
+                          <span>FOLLOWERS {profile.followers
+                          && profile.followers.length > 0
+                            ? profile.followers.length : '0'}
+                          </span>
+                        </div>
+                        <div className="profilebox-user__following">
+                          <span>FOLLOWING {profile.followings
+                          && profile.followings.length > 0
+                            ? profile.followings.length : '0'}
+                          </span>
+                        </div>
+                        <div className="profilebox-user__followers-btn">
+                          <button className="button is-success is-outlined">
+                            follow
+                          </button>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </section>
+            <section className="section">
+              <div className="container">
+                <div className="profile-tabs">
+                  <div className="tabs is-medium">
+                    <ul>
+                      <li
+                        className={isBio ? 'is-active' : ''}
+                        onClick={this.selectBio}
+                      ><a>Bio</a></li>
+                      <li><a>Articles</a></li>
+                      <li
+                        className={isBookmarks ? 'is-active' : ''}
+                        onClick={this.selectBookmarks}
+                      ><a>Bookmarks</a></li>
+                    </ul>
+                  </div>
+                  <div className="tabs-content">
+                    <div className="bio">
+                      {isBio ? <Bio profile={profile} /> : null}
+                      {
+                        isBookmarks
+                          ? <div className = "columns is-variable is-8 mt-3">
+                            <Bookmarks bookmarks={bookmarks} />
+                          </div>
+                          : null}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </section>
+          </div>
+        }
+      </div >
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  profile: state.userProfile.profile,
+  bookmarks: state.userProfile.bookmarks,
+});
+
+ProfilePage.propTypes = {
+  userProfileAction: PropTypes.func.isRequired,
```

Comment:
Great Job Victor. Please add default props for props that are not required.

Reply:
Thanks @ucheg6

Reply:
lol 🤣